### DISCUSSION TITLE
Remove the scroll wheel chat toggle or add an option to turn it off

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#4870] [Ideas]
Often the mouse pointer is placed near chat tabs to make it easier to switch between them and for some other reasons that are not relevant to the question. Because the scroll wheel is responsible for scrolling the chat itself, unfamiliar behavior unnoticeably changes the chat tab. 

You end up writing to a different chat room, especially if people in both chat rooms have familiar nicknames. I would like to point out that the focus while watching streams is more often on the video than on the chat.

I suggest removing this unusual behavior, which is not found almost anywhere in the interfaces, or adding an option to disable it to avoid unintentional errors.

What do you think?

**Comment by pajlada** [comment]
Chrome has this behaviour, the only thing we differ in is that we wrap, wheras Chrome does not
i3's status bar has this behaviour, again the only thing we differn in is that we wrap, wheras i3's status bar does not (at least not with my settings).

**Reply by pajlada** [reply]
I should also mention, I'm completely open to this being a setting so users can disable it.

**Reply by x9136** [reply]
> Chrome has this behaviour, the only thing we differ in is that we wrap, wheras Chrome does not

I downloaded Chrome, opened a few dozen tabs and I can't scroll through them using the scroll wheel like in Chatterino. Also I tried enabling this option (#scrollable-tabstrip) through experimental features, but it still doesn't work for unknown reasons.

Perhaps there are more differences after all. Does this work in your Chrome by default or did you customize it yourself? That would make a difference, because when you purposefully enable this option, it's a conscious choice. You may already be in the habit of switching tabs using the scroll wheel.

Does it work the same way as in Chatterino or, for example, do you have to hold some key on the keyboard to make the scroll wheel work in a different mode? If you have to hold a key, then that hold focuses the person exactly on changing the chat using the scroll wheel and prevents accidental switching.

**Reply by pajlada** [reply]
I very rarely use Chrome, but for me it works by default. Maybe it's a Linux-specific feature?
Either way, the feature will stay but I'm open to an setting to disable it.

**Comment by Mm2PL** [comment]
I totally don't get which behavior you're talking about.
